index.html

```diff
@@ -101,6 +101,31 @@ <h2 class="major">Paolo Mazza</h2>
           </div>
         </section>
 
+        <section id="one" class="wrapper spotlight style1">
+          <div class="inner">
+            <a href="#" class="image"
+              ><img src="images/profiles/martin.png" alt=""

```

Comment:
¡Crack! tu foto de perfil se ve leeeevemente ovalada, como lo que le había pasado a Sofi. Podés arreglarlo asignándole la clase `circle-image` a la imagen. 😁 
```suggestion
              ><img
                class="circle-image"
                src="images/profiles/martin.png"
                alt=""
            />
```